apps/ws/src/GameManager.ts

```diff
@@ -51,7 +51,7 @@ export class GameManager {
             console.error('Pending game not found?');
             return;
           }
-          if (user.userId === game.player1UserId) {
+          if (user.userId === game.player1.userId) {
```

Comment:
revert this as u revert the Game.ts logic

Reply:
reverted